EPPlus/Drawing/ExcelDrawingBorder.cs

```diff
@@ -170,11 +170,13 @@ private string TranslateLineStyleText(eLineStyle value)
         private eLineStyle TranslateLineStyle(string text)
         {
             switch (text)
-            {
+            {
+                case "":
+                    return (eLineStyle)Enum.Parse(typeof(eLineStyle), "solid", true);
                 case "dash":
                 case "dot":
                 case "dashDot":
-                case "solid":
+                case "solid":
```

Comment:
```C#
case "dash":
case "dot":
case "dashDot":
    return (eLineStyle)Enum.Parse(typeof(eLineStyle), text, true);
case "solid":
case "":
    return (eLineStyle)Enum.Parse(typeof(eLineStyle), "solid", true);
```

I think it should be refactored like this.
In case of `"dash", "dot", "dashDot"` it forward `text`
But in case of `"solid", ""` it forwards `"solid"` in the  way it forward `"Long"` when the `text`is from `"lgDash", "lgDashDot", ...`


Also, this switch case throw an exception in case of default.
I am not sure we should give a value for `""`.
If we want to merge to the master repo, I think it can break someone else codes.

I don't know if there is a way to transform `text` before being call by this method.
The method is private, So, I guess it is an internal behavior.

Reply:
I took a look to the whole file `TranslateLineStyle` does not seem to catch the exception.
I guess it is better to have a default behavior than throwing an exception.

Reply:
tnx for reviewing the code.

Reply:
tnx for reviewing the code.
I agree with you about the exception, but because this code is assumed as a external I tried to modify it's code as less as possible.
IMHO
the reason for `""` case is that, when some one who creates excel template uses a theme in his excel file excel does not store any  data about graph line styles in graphs in the file unless he changes the line style.
so in many cases user sees a solid line in the excel template but that `solid` style will not be passed to the library. so, we assumed that this is solid line by default.
but in the other hand maybe there is an invalid LineStyle passed to this function so it goes to the `default` section and throws the `exception`.

Reply:
Thanks for the explanation.
To summarize:
`""` means use default style
`not matching text` is not supported and throw exception

Do we plan to create a PR on the main repo?
If not then I am ok with this code.